### PR TITLE
{compiler}[system/system] ispc v1.16.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2619.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/e3345c647958ab1faca3f1d2bb42e20f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3545.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/91b583fb4607e94ac0fe00652b1472bf for a full test report.

File: easybuild/easyconfigs/i/ispc/ispc-1.16.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'Tarball'
+
+name = 'ispc'
+version = '1.16.0'
+
+homepage = 'http://ispc.github.io/ , https://github.com/ispc/ispc/'
```
Comment (Micket) [inline easybuild/easyconfigs/i/ispc/ispc-1.16.0.eb:6]:
```suggestion
homepage = 'https://ispc.github.io/ , https://github.com/ispc/ispc/'
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/c537a30fb85167e753d86e94cc691cd4 for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/fd67541a584bffa8ff3302d8455683b3 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/acd9e5a70b6f981a5b7aa1bb91517670 for a full test report.

**Pull request merged**